web-app/src/hoc/Layout/styled.js

```diff
@@ -4,12 +4,27 @@ export const Drawer = styled.div`
   background-color: ${props => props.theme.colours.unoBlue};
   color: ${props => props.theme.colours.white};
   position: fixed;
-  left: -160px;
+  left: -200px;
   top: 0;
   bottom: 0;
   width: 200px;
   transition: left 300ms;
   z-index: 10;
+  @media (min-width: 600px) {
+    left: -160px;
+  }
+
+  &::after {
+    @media (max-width: 768px) {
```

Comment:
A comment outside of the scope of the ticket, it'd be nice to have a constant files with the widths used for media queries.

Reply:
Nice point. I'm working on a global stylesheet for font sizes which will be held within styled components theme. I'll incorporate media sizes in to